nncf/openvino/quantization/ignored_patterns.py

```diff
@@ -108,3 +109,53 @@ def create_equal_logicalnot() -> GraphPattern:
 
     pattern.add_edge(equal_node, logical_not_node)
     return pattern
+
+
+@OPENVINO_IGNORED_PATTERNS.register(IgnoredPatternNames.SE_BLOCK)
+def create_se_block() -> GraphPattern:
+    pattern = GraphPattern()
+    any_node = pattern.add_node(
+        **{GraphPattern.LABEL_ATTR: "ANY", GraphPattern.METATYPE_ATTR: GraphPattern.NON_PATTERN_NODE_TYPE}
```

Comment:
Minor: I would change the label from `ANY` to `NON_PATTERN_NODE` to explicitly demonstrate the node is an outside pattern and not mixing up with `ANY_PATTERN_NODE_TYPE`

Reply:
Done